type/shot.js

```diff
@@ -81,7 +82,8 @@ const ShotType = new GraphQLObjectType({
     tags: { type: new GraphQLList(GraphQLString) },
     user: { type: User },
     team: { type: User },
-    images: { type: Images }
+    images: { type: Images },
+    attachments: { type: Attachment }
```

Comment:
Please remove this line for now, we can create separate issue for resolve this field.